readthedocs/doc_builder/base.py

```diff
@@ -48,7 +48,7 @@ def force(self, **__):
         log.info("Forcing a build")
         self._force = True
 
-    def build(self, id=None, **__):  # pylint: disable=redefined-builtin
+    def build(self):
```

Comment:
As far as i could tell, there were no child classes using any arguments on this method, and no calls with arguments. It seems safe to remove this, unless there is something hidden using these.